### PR TITLE
fixes issue #110 - DEV colum length to 15

### PR DESCRIPTION
**Pull request by pavan02**
Changed DEV column length increased to 15 characters and PROGRAM column is reduced by 10 characters...
This was the easiest thing I could think of....Dynamic length as suggested by you seems like its gonna be little messy....Let me know if you have better way to do it...

### CODE REVIEW

**Comment by raboof** [comment]
Thanks for taking the time to look into this! As mentioned in I'm not too keen on further reducing the width of the already-cramped PROGRAM column - would it be such a mess to find the maximum interface name width before drawing the rows?

File: src/cui.cpp [hunk 1]
```diff
@@ -334,29 +335,46 @@ void show_ncurses(Line *lines[], int nproc) {
   if (cols > PROGNAME_WIDTH)
     cols = PROGNAME_WIDTH;
 
-  proglen = cols - 65;
+
+ //issue #110 - dynamic length devicename
+ int devlen = MIN_COLUMN_WIDTH_DEV; int curlen;
```
Comment (raboof) [inline src/cui.cpp]: Looks neat! Perhaps move the logic to a private helper function that returns `devlen`?

File: src/cui.cpp [hunk 2]
```diff
@@ -175,15 +176,15 @@ static void mvaddstr_truncate_cmdline(int row, int col, const char *progname,
   }
 }
 
-void Line::show(int row, unsigned int proglen) {
+void Line::show(int row, unsigned int proglen,unsigned int devlen) {
```
Comment (raboof) [inline src/cui.cpp]: minor thing: I like having a space after the `,` ;)

**Comment by pavan02** [comment]
Sure..

**Comment by pavan02** [comment]
closed accidentally...

**Comment by raboof** [comment]
Thanks for your contribution!

**Pull request merged**